Misc/NEWS.d/next/Core and Builtins/2018-07-23-13-09-54.bpo-19072.Gc59GS.rst

```diff
@@ -0,0 +1,2 @@
+The :class:`classmethod` decorator now works correctly when it was wrapped by
```

Comment:
I'm not sure this NEWS entry is descriptive, so I'm open to suggestions.

Reply:
Isn't "when it **wraps** a descriptor" more correct?